File: ncms-extensions/ncms-js-converter/src/test/java/com/softmotions/ncms/js/TestRS.java
package com.softmotions.ncms.js;

import javax.annotation.Nonnull;

import com.softmotions.ncms.DbTestsFactory;

/**
 * @author Adamansky Anton (adamansky@softmotions.com)
 */
public class TestRS extends DbTestsFactory {

    @Override
    public Object[] createTests(@Nonnull String db) {
        return new Object[]{new _TestJsServiceRS(db)};
    }
}

File: ncms-engine/ncms-engine-tests/src/test/java/com/softmotions/ncms/rs/TestRS.kt
package com.softmotions.ncms.rs

import com.softmotions.ncms.DbTestsFactory


/**
 * @author Adamansky Anton (adamansky@softmotions.com)
 */
class TestRS : DbTestsFactory() {

    override fun createTests(db: String): Array<out Any> {
        return arrayOf(
                _TestAsmRS(db),
                _TestPageRS(db),
                _TestMediaRS(db),
                _TestMttRulesRS(db))
    }
}